spec/features/event_email.spec.rb

```diff
@@ -3,6 +3,26 @@
 feature 'Email Events', '' do
   fixtures :all
 
+  scenario 'When logged in as a requester' do
```

Comment:
Filename is wrong: s/spec/features/event_email.spec.rb/spec/features/event_email_spec.rb/